spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
+describe Menu do
+  it 'can initialize a constant of menu' do
+    menu = Menu.new
+  end
+
+  it 'holds a list of available items' do
+    menu = Menu.new
+    expect(menu.items).to include :chicken => 5.0
```

Comment:
This test ties you to a specific, hard-coded menu. https://github.com/nboyse/takeaway-challenge/blob/master/docs/review.md#open-closed-principle